plugins/woocommerce/client/legacy/js/frontend/woocommerce.js

```diff
@@ -119,4 +119,19 @@ jQuery( function( $ ) {
 			}
 		} );
 	} );
+
+	$( document ).ready( function() {
+		var $sorted_by_live_region = $( '.woocommerce-result-count[data-is-sorted-by="true"]' );
+
+		if ( $sorted_by_live_region.length > 0 ) {
+			var text = $sorted_by_live_region.first().html();
+			
+			$sorted_by_live_region.delay( 1000 ).queue( function() {
+				$( this )
+					.html( '' )
+					.html( text )
+					.dequeue();
+			} );
+		}
+	} );	
```

Comment:
Please can we rewrite this in vanilla JS? We are trying to reduce additional jQuery code being added to the codebase so we can remove it as a dependency sometime in the future. I don't think this particular block of code should be too difficult to rewrite and I'm happy to assist if needed.